homestar-runtime/Cargo.toml

```diff
@@ -3,7 +3,12 @@ name = "homestar-runtime"
 version = "0.1.0"
 description = "Homestar CLI"
 keywords = ["ipfs", "workflows", "ipld", "ipvm"]
-categories = ["workflow-engines", "distributed-systems", "runtimes", "networking"]
+categories = [
```

Comment:
FYI, did look into this and while cargo fmt doesn't do this officially, seems like there are extensions for vscode. I'm bringing-in one on my branch, called taplo that will take care of this locally, at least.

Reply:
Going to resolve this, as we'll use taplo moving forward.